pkg/network/usm/ebpf_gotls.go

```diff
@@ -103,31 +86,16 @@ type pid = uint32
 
 // goTLSProgram contains implementation for go-TLS.
 type goTLSProgram struct {
-	wg      sync.WaitGroup
-	done    chan struct{}
-	cfg     *config.Config
-	manager *manager.Manager
+	attacher  *uprobes.UprobeAttacher
+	inspector *GoTLSBinaryInspector
```

Comment:
Does `GoTLSBinaryInspector` need to be exported?

Reply:
It doesn't, changed.